packages/block-library/src/group/edit.js

```diff
@@ -48,23 +79,27 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 	return (
 		<>
 			<InspectorControls __experimentalGroup="advanced">
-				<SelectControl
-					label={ __( 'HTML element' ) }
-					options={ [
-						{ label: __( 'Default (<div>)' ), value: 'div' },
-						{ label: '<header>', value: 'header' },
-						{ label: '<main>', value: 'main' },
-						{ label: '<section>', value: 'section' },
-						{ label: '<article>', value: 'article' },
-						{ label: '<aside>', value: 'aside' },
-						{ label: '<footer>', value: 'footer' },
-					] }
-					value={ TagName }
-					onChange={ ( value ) =>
-						setAttributes( { tagName: value } )
-					}
-				/>
+				<div className="block-library-group-block-html-element-control">
+					<SelectControl
+						label={ __( 'HTML element' ) }
+						options={ [
+							{ label: __( 'Default (<div>)' ), value: 'div' },
+							{ label: '<header>', value: 'header' },
+							{ label: '<main>', value: 'main' },
+							{ label: '<section>', value: 'section' },
+							{ label: '<article>', value: 'article' },
+							{ label: '<aside>', value: 'aside' },
+							{ label: '<footer>', value: 'footer' },
+						] }
+						value={ TagName }
+						onChange={ ( value ) =>
+							setAttributes( { tagName: value } )
+						}
+						help={ getHTMLElementMessage( TagName ) }
```

Comment:
Just accessing an object like `messages[ TagName ]` would work well and we will not need to make an extra function call. If `help` is undefined, it will not show anything. We just need to rename the `object` `messages` something more contextual like `htmlElementMessages` or something like that.

Reply:
Done